engine/utils/exec.go

```diff
@@ -3,11 +3,13 @@ package utils
 import (
 	"bytes"
 	"os/exec"
+	"syscall"
 )
 
 // ExecExternal is a helper function to run an external command
 func ExecExternal(dir string,name string, arg ...string)  (outStr string, errStr string, err error) {
 	cmd := exec.Command(name, arg...)
+	cmd.SysProcAttr = &syscall.SysProcAttr{Setsid: true}
```

Comment:
Could you add a comment specifying what is the purpose of this line?

Reply:
Documented what is the intended effect (create a new session without a terminal attached, thus providing a fail mechanism if human input is required)